easybuild/easyblocks/c/cuda.py

```diff
@@ -68,13 +68,23 @@ def install_step(self):
         if LooseVersion(self.version) <= LooseVersion("5"):
             install_script = "install-linux.pl"
             install_script_path = os.path.join(self.builddir, install_script)
```

Comment:
@vanzod As far as I can tell, `install_script_path` is totally unused in the easyblock, so please remove it